### PR TITLE
Stacks: allow fasta format for process_radtags

### PR DESCRIPTION
**Pull request by abretaud**
Just a PR to allow fasta input to process_radtags.

### CODE REVIEW

File: tools/stacks/stacks_procrad.xml [hunk 1]
```diff
@@ -12,6 +12,9 @@
             #if $input_type.input_single.ext == "fastqsanger":
                 #set $ext = ".fq"
                 #set inputype = "fastq"
+            #else if $input_type.input_single.ext == "fasta":
```
Comment (jmchilton) [inline tools/stacks/stacks_procrad.xml]: http://planemo.readthedocs.io/en/latest/writing_how_do_i.html#check-input-type-in-command-blocks

I think you should use: ``$input_type.input_single.is_of_type('fasta')`` instead.

**Comment by abretaud** [comment]
Oh yes thanks, done!

**Comment by yhoogstrate** [comment]
Would you mind adding a simple test case for the fasta format?

**Comment by abretaud** [comment]
Hmm, well, I don't know why, I was sure stacks could take fasta as input, but it seems like it's not the case.
Still, this PR contains the change to is_of_type()
Sorry for the false hope

**Comment by bgruening** [comment]
@abretaud can you please increase the verison number?

Thanks.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**